### PR TITLE
Unread Messages Bubble 

### LINKED ISSUE #67: Message notification bubble
show a notification bubble for new unread messages

screenshot

### PR DESCRIPTION
**Pull request by lennet**
fixes #67

### CODE REVIEW

File: CriticalMass/ChatNavigationButton.swift [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+//
+//  Created by Leonard Thomas on 4/10/19.
+//
+
+import UIKit
+
+class ChatNavigationButton: CustomButton {
+    let unreadLabel = UILabel()
+
+    init(chatManager: ChatManager) {
```
Comment (mltbnz) [inline CriticalMass/ChatNavigationButton.swift]: I would argue that the Button should not know about where the number that it should display is coming from. Meaning maybe there could be a controller kind of object that sets the number. Then the button would only disply it.

**Comment by lennet** [comment]
Bildschirmfoto 2019-04-11 um 18 13 52
Bildschirmfoto 2019-04-11 um 18 14 08

**Comment by zutrinken** [comment]
@lennet Screenshot looks perfect. But did you forget to push? Still looks the same on my machine:

Bildschirmfoto 2019-04-11 um 19 52 52

**Comment by lennet** [comment]
Yes, I did  😄

**Comment by headione** [comment]
Great @lennet !! One thing, could we please add the changes to our changelog?

In this case I would add `Unread Messages Bubble` here. It's just better for the overwiew later :) Our changelog follows the convention of keepachangelog: https://keepachangelog.com/en/1.0.0/ Thanks!

**Pull request merged**